Conf/MigrationPackage/migration.sh

```diff
@@ -1,4 +1,37 @@
 #!/bin/bash
 
-# TODO: Add migration logic
-while true; do logger "migration"; sleep 1; done;
+while true;
+do
+  GOOGLE_SANTA_ACTIVATED="$(/usr/bin/systemextensionsctl list com.apple.system_extension.endpoint_security |
+                            /usr/bin/grep -E '^.+EQHXZ8M8AV.+com\.google\.santa\.daemon.+activated.+$')"
+  if [ -z "${GOOGLE_SANTA_ACTIVATED}" ]; then
+    break;
+  else
+    sleep 1
```

Comment:
Probably doesn't need to be this tight a loop, 10s or even 30s would be plenty quick enough

Reply:
good call, upped it to 10s